zend_parse_arg_impl.go

```diff
@@ -49,19 +53,8 @@ func zendParseArgAsString(value any) (string, error) {
 	var str string
 
 	switch v := value.(type) {
-	case string:
-		str = v
-	case int, int64:
-		str = fmt.Sprintf("%v", v)
-	case float64:
-		str = floatToString(v)
-	case bool:
-		// return "1" for true and an empty string("") for false
-		if v {
-			str = "1"
-		} else {
-			str = ""
-		}
+	case string, int, int64, float64, bool:
+		return ConvertToString(value)
```

Comment:
It looks `ConvertToString` function cover those `nil` and `toStringAble` cases below. So... I think  it would be better merge `nil` and `toStringAble` cases into above case.